### ISSUE TITLE
Ubiquitous sections

### ISSUE THREAD

**Issue by manuelzane** [bedita/bedita#944]
When opening a section which contains other sections in a browser tab, then when opening one of his subsections in another tab and move this section to another position in the tree (section) and save, then return to the previous tab and save, will save the subsection both in the new position and the old.

**Comment by fquffio** [comment]
In this scenario, parent section's form still has information about the child section. Thus, when submitting, the `reorder` form parameter holds an ID of a section that is no longer a descendant.

In `AreasController::saveSection()` this information is used to reorder section's children's priority. When `Tree::appendChild()` is invoked, it re-adds the formerly descendant section to the subtree, making it ubiquitous.

Long story short, `Tree::appendChild()` should perform additional validation on the parameters.

@manuelzane @batopa @stefanorosanelli @qwerg What is the expected behavior in such a (weird) scenario? Should the child section's position be restored? Is it supposed to stay where it has been placed? Should an error be raised? Should we explicitely blame Simona? 😂

**Comment by stefanorosanelli** [comment]
I would say: if a subsection is present in section's form but on save action has been moved -> save operations should fail with a message and through page reload current situation is restored

**Comment by fquffio** [comment]
@manuelzane @stefanorosanelli This issue should have been addressed by now. Could you please test and close the issue when everything's OK?

**Comment by manuelzane** [comment]
@fquffio the behavior seems ok, but the alert disappears too quickly and it's unreadable.

**Comment by fquffio** [comment]
@manuelzane Have you pulled from the repo recently? The alert-disappearing-too-quickly issue should have been addressed by 14e18b48.
